src/types/gqlTypes.ts

```diff
@@ -42994,12 +42994,17 @@ export function useGetRootPermissionsLazyQuery(
   >(GetRootPermissionsDocument, options);
 }
 export function useGetRootPermissionsSuspenseQuery(
-  baseOptions?: Apollo.SuspenseQueryHookOptions<
-    GetRootPermissionsQuery,
-    GetRootPermissionsQueryVariables
-  >
-) {
-  const options = { ...defaultOptions, ...baseOptions };
+  baseOptions?:
+    | Apollo.SkipToken
```

Comment:
this file contains type changes for `skipToken`  https://www.apollographql.com/docs/react/api/react/hooks it seems to be only for suspense queries which we don't use much